modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -102,10 +107,108 @@ endif::openshift-origin[]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+ifndef::openshift-origin[]
+  networkType: OpenShiftSDN
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+  networkType: OVNKubernetes
+endif::openshift-origin[]
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  ibmcloud:
+    region: eu-gb <1>
+    resourceGroupName: eu-gb-example-network-rg
+    vpcName: eu-gb-example-network-1 <6>
+    controlPlaneSubnets: <7>
+      - eu-gb-example-network-1-cp-eu-gb-1
+      - eu-gb-example-network-1-cp-eu-gb-2
+      - eu-gb-example-network-1-cp-eu-gb-3
+    computeSubnets: <8>
+      - eu-gb-example-network-1-compute-eu-gb-1
+      - eu-gb-example-network-1-compute-eu-gb-2
+      - eu-gb-example-network-1-compute-eu-gb-3
+credentialsMode: Manual
+publish: External
+pullSecret: '{"auths": ...}' <1>
+ifndef::openshift-origin[]
+fips: false <9>
+sshKey: ssh-ed25519 AAAA... <10>
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+sshKey: ssh-ed25519 AAAA... <9>
+endif::openshift-origin[]
+capabilities:
+  baselineCapabilitySet: None
+  additionalEnabledCapabilities:
+  - openshift-samples
+----
+<1> Required. The installation program prompts you for this value.
+<2> If you do not provide these parameters and values, the installation program provides the default value.
+<3> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
+====
+<5> The machine CIDR must contain the subnets for the compute machines and control plane machines.
+<6> Specify the name of an existing VPC.
+<7> Specify the name of the existing subnets to deploy the control plane machines to. The subnets must belong to the VPC that you specified. Specify a subnet for each availability zone in the region.
```

Comment:
suggest add 
resourceGroupName: the resource group name of the existing VPC, and the cluster will be deployed in the resource goup.